### PR TITLE
style: explore update UI

### PR DESCRIPTION
**Pull request by RominaMarchetti**
This PR contains cosmetic changes in order to align the Discover UI with the latest changes in the backpack and upcoming account/wallet section. Plus, it includes a legibility improvement in the navigation bar: bigger font size and bigger icons.

**Nav bar**

Screenshot 2023-06-13 at 14 56 45

**Discover sections**
- Discover tab bar updated adding a title, a background container and bigger buttons now aligned on the left
- New background
- Next/forward buttons containers added
- Jump In button font style updated to Semi-Bold

Screenshot 2023-06-13 at 14 57 26

### CODE REVIEW

**Review by kwssbocian — APPROVED** [review]
✔️NavBar has been updated and features bigger font size, icons and less padding away from navbar borders.
✔️Discover tab bar now features a title, slightly darker background and the icons are aligned left.
✔️Containers for next/forward buttons are present.
✔️Font style for the Jump In button has been changed.
All altered UI elements work correctly.

**Review by AjimenezDCL — APPROVED** [review]

File: unity-renderer/Assets/Textures/UI/Common/FavoriteIcon.png.meta [hunk 1]
```diff
@@ -6,7 +6,7 @@ TextureImporter:
   serializedVersion: 12
   mipmaps:
     mipMapMode: 0
-    enableMipMap: 1
+    enableMipMap: 0
     sRGBTexture: 1
     linearTexture: 0
```
Comment (popuz) [inline unity-renderer/Assets/Textures/UI/Common/FavoriteIcon.png.meta:11]: strange that this .meta file remained even that you removed original .png. Maybe this .meta-file is the reason why I have both files when pull you branch
image

File: unity-renderer/Assets/Textures/UI/Common/TexturesUI.spriteatlas [hunk 2]
```diff
@@ -443,7 +443,7 @@ SpriteAtlas:
   - SpeakerOn
   - ContainerRadius6
   - VoiceChatCross
-  - DecentralandIsologotipo68
+  - DCLLogo
   - TopRect12
```
Comment (popuz) [inline unity-renderer/Assets/Textures/UI/Common/TexturesUI.spriteatlas:447]: it is so difficult to find this Atlas in the one big folder of assets. I would suggest add sub-folders for better visibility and clarity. For example .spriteatlas can be in the root UI/Common, while all atlased textures can be in sub-folder(s) depending on their grouping in the UI (as I see, not all of them are even so "Common" - EmotesOn/Off, VoiceChat. Favorites and etc)

**Review by popuz — APPROVED** [review]
approved with small folder structure comment

**Pull request merged**